docs/en/glossary/glossary.asciidoc

```diff
@@ -550,6 +572,21 @@ nodes handling the user requests.
 //Source: Cloud
 endif::cloud-terms[]
 ifdef::elasticsearch-terms[]
+[[glossary-query]] query ::
+
+A query is the basic component of a search. A search can be defined by one or
```

Comment:
These feel more verbose than I'd expect for a glossary entry, but I guess it doesn't matter too much.

Reply:
Agree. I think a rewrite is probably in order. Query isn't actually defined here IMO.

I opened elastic/elasticsearch#40643 to address that separately. I'm planning on doing a general query overhaul shortly. That issue will likely be a part of that work effort.